docs/src/languages/elixir.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Elixir"
+weight: 2
+description: Platform.sh supports building and deploying applications written in Elixir. There is no default flavor for the build phase, but you can define it explicitly in your build hook. Platform.sh Elixir images support both committed dependencies and download-on-demand. The underlying Erlang version is 22.0.7.
```

Comment:
Does the description get shown on the page as well?

Reply:
Yes, with the `{{< description >}}` shortcode below it.